openfisca_us/parameters/gov/states/ny/tax/income/supplemental/agi_limit/max.yaml

```diff
@@ -0,0 +1,11 @@
+description: Filers with AGI over this limit pay a flat rate on taxable income.
+SINGLE:
+  2022-01-01: 25_000_000
```

Comment:
I don't see this value in the linked reference, could you point to a search term I can use?

Reply:
Ah it was the wrong one - should point to 601 instead. Fixed.